sql/mysql/mysqlcheck/mysqlcheck.go

```diff
@@ -188,3 +182,16 @@ func (d *DataDepend) addColumn(p *sqlcheck.Pass, change *sqlcheck.Change, t *sch
 	}
 	return
 }
+
+func init() {
+	sqlcheck.Register(mysql.DriverName, func(r *schemahcl.Resource) (sqlcheck.Analyzer, error) {
+		az, err := NewDataDepend(r)
+		if err != nil {
+			return nil, err
+		}
+		return sqlcheck.Analyzers{
+			destructive.New(destructive.Options{}),
+			az,
+		}, nil
+	})
```

Comment:
This means runtime errors instead of compile time, right?

Reply:
Both happen on initializations because an error is returned in the case of an invalid configuration.